tools/docker/docker-compose.yml

```diff
@@ -35,5 +50,45 @@ services:
         depends_on:
             - db
 
+    char:
+        image: "rathena:local"
+        container_name: "rathena-char"
+        command: "-c ./char-server"
```

Comment:
these should probably be using the absolute path just for clarity's sake 
`command: "-c /rathena/char-server"`